corehq/apps/reports/standard/monitoring.py

```diff
@@ -1461,8 +1470,18 @@ def get_admins_and_demo_users(self):
     @property
     def users_to_iterate(self):
         if toggles.EMWF_WORKER_ACTIVITY_REPORT.enabled(self.request.domain):
-            user_query = EMWF.user_es_query(self.domain, self.request.GET.getlist(EMWF.slug))
-            return util.get_simplified_users(user_query)
+            user_query = LocationRestrictedMobileWorkerFilter.user_es_query(
+                self.domain, self.request.GET.getlist(LocationRestrictedMobileWorkerFilter.slug)
+            )
+            users = util.get_simplified_users(user_query)
```

Comment:
could you move these 4 lines to an else clause below? seems like `users` can be overridden below and we could save a hit to es by not running these